column_chunk.go

```diff
@@ -1,9 +1,15 @@
 package parquet
 
 import (
+	"errors"
 	"io"
 )
 
+var (
+	ErrMissingColumnIndex = errors.New("missing column index")
+	ErrMissingOffsetIndex = errors.New("missing offset index")
```

Comment:
Could we change the wording slightly to something like "page index skipped on file open" just something to suggest that the index isn't missing but was intentionally not read?

Reply:
Hm I don't think that would be correct though. Because of the lazy load feature, we would now try to read the index even if `SkipPageIndex` was used when opening a file. Is this breaking your use case?